### PR TITLE
1020 reduce error traces in bmcweb

### PR DESCRIPTION
**Pull request by geissonator**
One commit in this PR is upstream at https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/48795/

The other two commits are for code that we only have in downstream.

### CODE REVIEW

File: redfish-core/lib/chassis.hpp [hunk 1]
```diff
@@ -136,8 +136,8 @@ inline void getPhysicalSecurityData(std::shared_ptr<bmcweb::AsyncResp> aResp)
             {
                 // do not add err msg in redfish response, because this is not
                 //     mandatory property
-                BMCWEB_LOG_ERROR << "DBUS error: no matched iface " << ec
-                                 << "\n";
+                BMCWEB_LOG_INFO << "DBUS error: no matched iface " << ec
```
Comment (gtmills) [inline redfish-core/lib/chassis.hpp:139]: Since we don't have /xyz/openbmc_project/Intrusion it is hitting here

Comment (gtmills) [reply redfish-core/lib/chassis.hpp:139]: I don't think I see this one upstream.. Can we upstream this one?

**Review by gtmills — APPROVED** [review]
Looks good, please just keep pushing those upstream

File: redfish-core/lib/chassis.hpp (hunk 1 above)
Comment (geissonator) [reply redfish-core/lib/chassis.hpp:139]: Hmm, I have this upstream at https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/48795/2/redfish-core/lib/chassis.hpp. Is that what you're asking or more about the intrusion sensor function?

Comment (gtmills) [reply redfish-core/lib/chassis.hpp:139]: Got it. 
Nah, I am good. I missed it.

**Review by manojkiraneda — APPROVED** [review]

**Pull request merged**